### PR TITLE
docs(agw): Sentry Documentation

### PR DESCRIPTION
**Pull request by jaredmullane**
Signed-off-by: Jared Mullane <jmullane@fb.com>

## Summary

Updated Sentry documentation with instructions on utilizing the Linux Foundation account. 

## Test Plan

cd ${MAGMA_ROOT}/docs && make precommit

cd ${MAGMA_ROOT}/docs && make precommit_fix

### CODE REVIEW

**Comment by themarwhal** [comment]
Should we also update this doc as well? `/docs/readmes/lte/configure_sentry.md`

@jaredmullane Hey Jared! Please run the doc precommit fix command as described here: https://magma.github.io/magma/docs/next/docs/docs_overview#precommit

And @quentinDERORY, if you could update this doc once the agw release job is migrated, that would be very helpful. (FYI @jaredmullane We are in the middle of a migration from CircleCI to Github Action for this release job)

File: docs/docusaurus/versioned_docs/version-1.6.0/lte/configure_sentry.md [hunk 1]
```diff
@@ -66,3 +81,5 @@ do
     sentry-cli upload-dif --log-level=info --org="$ORG" --project="$PROJECT" "$EXEC"
 done
 ```
+
+
```
Comment (electronjoe) [inline docs/docusaurus/versioned_docs/version-1.6.0/lte/configure_sentry.md]: Nit / question - are two newlines necessary at the end of the doc?

**Review by electronjoe — APPROVED** [review]
Thanks for the documentation improvements @jaredmullane!

**Comment by jaredmullane** [comment]
@themarwhal / @electronjoe updated
Screen Shot 2021-12-06 at 3 45 12 PM
.

**Review by hcgatewood — APPROVED** [review]
💯💯

**Comment by themarwhal** [comment]
merge conflict :'( cc @jaredmullane

**Comment by jaredmullane** [comment]
@themarwhal should hopefully be good now *fingers crossed*

**Pull request merged**